scripts/deployIdentityMlanaget.js

```diff
@@ -0,0 +1,14 @@
+const { JsonRpcProvider } = require('ethers/providers');
+
+const { ContractFactory, Wallet } = require('ethers');
+const { abi, bytecode } = require('../packages/proxyIdentity/build/contracts/IdentityManager.json');
+
+const deployer = new Wallet(
+  '9d69bf3c7bda0fc771a59818e5a7f894f3b4fbd3114891db3d4de7f46aa71ca8',
+  new JsonRpcProvider('https://volta-rpc-vkn5r5zx4ke71f9hcu0c.energyweb.org/'),
```

Comment:
IMO, we should just use volta-rpc.energyweb.org here. I know we are using it publicly in Switchboard, but I think it is supposed to be an internal endpoint so better not to have it checked into ew-did-registry code